standards/core/lightpush.md

```diff
@@ -0,0 +1,88 @@
+---
+title: WAKU2-LIGHTPUSH
+name: Waku v2 Light Push
+status: draft
+editor: Zoltan Nagy <zoltan@status.im> 
+contributors: 
+  - Hanno Cornelius <hanno@status.im>
+  - Daniel Kaiser <danielkaiser@status.im>
+  - Oskar Thorén <oskarth@titanproxy.com>
+---
+
+previous version: `/vac/waku/lightpush/2.0.0-beta1` [19/WAKU2-LIGHTPUSH](https://rfc.vac.dev/waku/standards/core/19/lightpush)
+
+---
+**Protocol identifier**: `/vac/waku/lightpush/3.0.0`
+
+## Motivation and Goals
+
+Light nodes with short connection windows and limited bandwidth wish to push messages to other nodes in the Waku network to request message services.
+A common use case is to request that service node to publish the message to a `11/WAKU2-RELAY` pubsub topic.
+Additionally, there is sometimes a need for confirmation that a message has been received "by the network"
+(here, at least one node).
+
+`WAKU2-LIGHTPUSH` is a request/response protocol for this.
+
+## Payloads
+
+```protobuf
+syntax = "proto3";
+
+message LightPushRequest {
+        string request_id = 1;
+    uin32 request_type = 10;  // reserved for future request type selection, currently it is always RELAY (0)
+    string topic = 20;
```

Comment:
```suggestion
    optional string pubsub_topic = 20;
```
We should be explicit about this being a `pubsub_topic`. Note that making the field `optional` here doesn't mean that it is optional in the spec - it will simply allow the service node to verify that a `pubsub_topic` has been explicitly set by the client. Otherwise it will by default interpret any message withouth `pubsub_topic` as being directed at the empty string pubsub topic.

Reply:
Right, thank you.